tests/unit/test_utils.py

```diff
@@ -37,7 +37,7 @@ class MyUser(AbstractBaseUser):
     USERNAME_FIELD = 'identity'
 
     class Meta:
-        abstract = True
+        abstract = False
```

Comment:
I'm not sure about this change, again maybe it is ok here but it seems like a breaking change in Django and I can't see it in the release notes.